doc/classes/Transform3D.xml

```diff
@@ -21,6 +21,7 @@
 			<return type="Transform3D" />
 			<description>
 				Constructs a [Transform3D] identical to the [constant IDENTITY].
+				[b]Note:[/b]: In C#, constructs a zero-initialized [Transform3D] with all components set to [code]0.0[/code]. Use [constant IDENTITY] instead.
```

Comment:
```suggestion
				[b]Note:[/b]: In C#, this constructs a [Transform3D] with all of its components set to [code]0.0[/code].
```

Reply:
```suggestion
				[b]Note:[/b]: In C#, this constructs a [Transform3D] with an [member origin] set to [Constant Vector3.ZERO] and a [member basis] with all of its components set to [constant Vector3.ZERO].


```
This is the most technically correct way to describe the components of Transform3D, if that's the goal.